src/google/protobuf/unittest_import_public.proto

```diff
@@ -35,6 +35,7 @@ syntax = "proto2";
 package protobuf_unittest_import;
 
 option java_package = "com.google.protobuf.test";
+option csharp_namespace = "Google.ProtocolBuffers.TestProtos";
```

Comment:
It makes me wonder why for some test protos, java uses 
com.google.protobuf.test, for others it uses com.google.protobuf and for some it doesn't define the package at all. Do we really need csharp_namespace for all the test protos? It's for tests, rights, so arguably we don't care much what namespace we end up with with the generated protos.
Feel free to ignore this comment.